lib/meta_inspector/request.rb

```diff
@@ -28,7 +28,7 @@ def initialize(initial_url, options = {})
     delegate :url => :@url
 
     def read
-      response.body if response
+      response.body.gsub(/\0/,"") if response
```

Comment:
Yep good catch :)

According to https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code `String#tr` is faster than `String#gsub`.

Changing to `String#tr`